docs/static/State.md

```diff
@@ -3,37 +3,50 @@ Nostrum keeps track of the state that your bot can see. This state is updated
 based on events from the WS connection. Most of the cache modules support only
 simple interactions with the cache. Feel free to suggest additional functionality.
 
+## Pluggable caches
+
+The default caches supplied by nostrum should work for most of your needs, but
+all of them caches can be exchanged for your own implementations. For this,
```

Comment:
```suggestion
all of the caches can be exchanged for your own implementations. For this,
```

Very minor grammatical error. Otherwise this looks good to me.